package.json

```diff
@@ -125,6 +127,7 @@
     "babel-plugin-transform-inline-environment-variables": "0.5.0-alpha.42",
     "babel-preset-react-app": "10.0.1",
     "bundlewatch": "0.3.3",
+    "contentful-migrate": "^0.17.0",
```

Comment:
i wouldnt add it at the root, similar to how we should not have squidex or algolia deps anywhere outside of the squidex/algolia package

Reply:
Done.